### PR TITLE
add intro and example of stain norm docs

### PR DESCRIPTION
**Pull request by nicolebussola**
## Description

Update documentation with stain normalization methods (Macenko and Reinhard)

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Documentation

## Issues Fixed or Closed by This PR

* Fixes:

## Checklist

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [CONTRIBUTING](https://github.com/histolab/histolab/blob/master/CONTRIBUTING.md) document.
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by ernestoarbitrio** [comment]
IMHO I'd add an example of the usage of the stain normalizer

**Comment by alessiamarcolini** [comment]
Hi @nicolebussola I agree with @ernestoarbitrio, a usage example would be great!

File: docs/api/stain_normalizer.rst [hunk 1]
```diff
@@ -1,6 +1,22 @@
 Stain Normalizer
 ================
 
+Digital pathology images can present strong color differences due to diverse acquisition techniques (e.g., scanners, laboratory equipment and procedures)
+The ``stain_normalizer`` module collects methods for to transfer the staining style of an image (target) to another image (source).
```
Comment (alessiamarcolini) [inline docs/api/stain_normalizer.rst:5]:
```suggestion
The ``stain_normalizer`` module collects methods to transfer the staining style of an image (target) to another image (source).
```

**Review by alessiamarcolini — CHANGES_REQUESTED** [review]

**Comment by ernestoarbitrio** [comment]
> @nicolebussola ehm the docs are not rendering the API docs and the example (you can see the readthedocs check), can you fix this please?

I can see the page @alessiamarcolini  https://histolab--506.org.readthedocs.build/en/506/api/stain_normalizer.html

**Comment by alessiamarcolini** [comment]
@ernestoarbitrio yes the page does exist, **the docs are not rendering the API docs and the example**

**Comment by ernestoarbitrio** [comment]
> @ernestoarbitrio yes the page does exist, **the docs are not rendering the API docs and the example**

what do you see? I do see this:
image
is ☝🏽 wrong?

File: docs/api/stain_normalizer.rst [hunk 2]
```diff
@@ -10,3 +27,53 @@ Stain Normalizer
     :inherited-members:
 
 .. toctree::
+
+
+Usage
+----------
+
+The stain normalization methods can be used to tranfer the staining's appereance of any histology image. First, we need to establish the image to be used as style reference, and the image to be transformed accordigly:
```
Comment (alessiamarcolini) [inline docs/api/stain_normalizer.rst:35]:
```suggestion
The stain normalization methods can be used to tranfser the staining's appearance of any histology image. First, we need to establish the image to be used as a style reference, and the image to be transformed accordingly:
```

File: docs/api/stain_normalizer.rst [hunk 3]
```diff
@@ -10,3 +27,53 @@ Stain Normalizer
...
+   source_image
+
+.. figure:: https://user-images.githubusercontent.com/31658006/212924179-a85573b6-1bb3-4f9b-a8ab-00a26b1d652e.png
+   :align: center
+   :width: 200
+   :figclass: align-center
+
+
+
+The chosen stain normalization method must be then fit on the target image and then applied to the source image:
```
Comment (alessiamarcolini) [inline docs/api/stain_normalizer.rst:61]:
```suggestion
The chosen stain normalization method must be first fit on the target image and then applied to the source image:
```

**Review by alessiamarcolini — CHANGES_REQUESTED** [review]

File: docs/api/stain_normalizer.rst [hunk 4]
```diff
@@ -10,3 +27,53 @@ Stain Normalizer
     :inherited-members:
 
 .. toctree::
+
+
+Usage
+----------
+
+The stain normalization methods can be used to tranfser the staining's appearance of any histology image. First, we need to establish the image to be used as a style reference, and the image to be transformed accordingly:
```
Comment (alessiamarcolini) [inline docs/api/stain_normalizer.rst:35]:
```suggestion
The stain normalization methods can be used to transfer the staining's appearance of any histology image. First, we need to establish the image to be used as a style reference, and the image to be transformed accordingly:
```

**Review by alessiamarcolini — CHANGES_REQUESTED** [review]
ouch typo for the correction of the typo, my bad

**Comment by hannah413** [comment]
when will these features be made available in a conda package? i'd implement them in my pipelines today if they were available.

**Comment by alessiamarcolini** [comment]
Hi @hannah413 
this PR only introduces changes to the documentation, the features are already included in our latest release :)

**Comment by hannah413** [comment]
@alessiamarcolini hmm. histolab 0.5.1 is the most up to date version available on conda. i have histolab 0.5.1 installed and can run tissue extraction with it, but when i try to import a stain normalizer like this it says the module doesn't exist:

---------------------------------------------------------------------------
```
ModuleNotFoundError                       Traceback (most recent call last)
Cell In[1], line 1
----> 1 from histolab.stain_normalizer import ReinhardStainNormalizer

ModuleNotFoundError: No module named 'histolab.stain_normalizer'
```

**Comment by alessiamarcolini** [comment]
@hannah413 it looks like the latest 0.6.0 release is only available on PyPI for some reason 🤔 on it!

**Comment by hannah413** [comment]
@alessiamarcolini thank you!!!

**Pull request merged**